src/tests/test_api.py

```diff
@@ -31,7 +31,7 @@ class TestAPIShutdown(TestAPIProto, TestProcessShutdown):
     """Separate test case for API command 'shutdown'"""
     def test_shutdown(self):
         """Shutdown the pybitmessage"""
-        self.assertEquals(self.api.shutdown(), 'done')
+        self.assertEqual(self.api.shutdown(), 'done')
```

Comment:
oops, it's really deprecated since 2.7

Reply:
The source code still in python 2.7, and the method alos deprecates in 2.. So, should i replace with assertEquals or leave it as it is.

Reply:
Yes, replace. I agree (`assertEquals`, which I used is already deprecated), just noticed when you wrote. That's what my comment about. Sorry if it's not clear enough.

Reply:
No problem! :)

Reply:
No problem! :)